src/test/scala/com.lightbend.paradox/unidoc/UnidocPluginSpec.scala

```diff
@@ -4,5 +4,135 @@
 
 package com.lightbend.paradox.unidoc
 
-class UnidocPluginSpec {
+import com.lightbend.paradox.markdown.Writer
+
+class UnidocPluginSpec extends MarkdownBaseSpec {
+  val rootPackage = "akka"
+
+  val allClasses = Array(
```

Comment:
In production, classes are loaded by sbt, but sbt features are not accessible from the test code. So using hard-coded list here.